cypress/e2e/po/detail/provisioning.cattle.io.cluster/cluster-detail.po.ts

```diff
@@ -21,11 +21,51 @@ export default abstract class ClusterManagerDetailPagePo extends PagePo {
     return this.self().find('.primaryheader h1');
   }
 
+  logsContainer() {
+    return this.self().find('.logs-container');
+  }
```

Comment:
a PO needs to be created for this such as`ProvisioningLogPo` and we should access `logsContainer()` from there